fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -322,5 +322,10 @@ def self.hide_loading_indicator
         @require_fastlane_spinner.success
       end
     end
+
+    # checks if a given path is an executable file
+    def self.executable?(cmd_path)
+      cmd_path && (File.executable?(cmd_path) || (Helper.is_windows? && File.exist?(cmd_path))) && !File.directory?(cmd_path)
```

Comment:
Last nitpick: I think the call should be `Helper.windows?`

Reply:
`is_windows?` is an alias for backwards compatibility

Reply:
Oh good point. 
I have to reorder that file, getting to confusing.